pkg/build/buildkit/buildx.go

```diff
@@ -88,17 +89,29 @@ func (b *Builder) BuildInvocationImage(ctx context.Context, manifest *manifest.M
 	}
 
 	currentSession := []session.Attachable{authprovider.NewDockerAuthProvider(dockerconfig.LoadDefaultConfigFile(b.Err))}
+
 	ssh, err := buildflags.ParseSSHSpecs(opts.SSH)
 	if err != nil {
 		return span.Errorf("error parsing the --ssh flags: %w", err)
 	}
-	currentSession = append(currentSession, ssh)
+
+	pbssh, err := pb.CreateSSH(ssh)
+	if err != nil {
+		return span.Errorf("error parsing the --ssh flags: %w", err)
```

Comment:
```suggestion
		return span.Errorf("error creating ssh ", err)
```
Some error that isn't the same as the previous error.